### ISSUE TITLE
Test suite unavailable on Debian Jessie

### ISSUE THREAD

**Issue by lkarsten** [Dridi/cashpack#1]
When building ff8d3be on Debian Jessie, the tests are not possible to run.

```
lkarsten@lima:~/work/cashpack/build$ cmake ..
-- Detecting astyle program
-- Detecting astyle program - missing
-- Check for working hexdumps
-- Check for working hexdumps - broken
CMake Warning at tst/CMakeLists.txt:70 (message):
  The test suite can't be run:

  /home/lkarsten/work/cashpack/tst/hexcheck: 26: set: Illegal option -o
  pipefail



-- Configuring done
-- Generating done
-- Build files have been written to: /home/lkarsten/work/cashpack/build
```

**Comment by Dridi** [comment]
Thank you for the report, this is a portability issue but the `pipefail` option is not critical so I will remove it.

Also found by running the test suite with `dash(1)`:
```
$ echo $((2#00101010))
dash: 2: arithmetic expression: expecting EOF: "2#00101010"
```

**Comment by Dridi** [comment]
Is it still failing on Jessie? It should have been fixed by 58e4176.

**Comment by Dridi** [comment]
Timing out.
